Sources/Core/Formatter/MultilineFormatter.swift

```diff
@@ -39,10 +39,31 @@ class MultilineFormatter: PrettyFormatter {
         if fields.count == 1, let field = fields.first {
             body = "\(field.0): \(field.1)"
         } else {
-            let contents = fields.map { "\($0): \($1)" }.joined(separator: ",\n")
-            body = contents.indentTail(size: prefix.count)
+            body = fields.map(indentInsertedKeyValueString(_:))
+                .joined(separator: ",\n")
+                .indentTail(size: prefix.count)
         }
 
         return prefix + body + ")"
     }
+
+    /// transform key value pair to single string inserted indent according to key
+    ///
+    /// ex:
+    /// ("owner", """
+    /// Owner(name: "Nanachi",
+    ///       age: 4)
+    /// """)
+    ///
+    /// goes to
+    ///
+    /// owner: Owner(name: "Nanachi",
+    ///              age: 4))
+    ///
+    /// - Parameter source: key value pair
+    private func indentInsertedKeyValueString(_ source: (String, String)) -> String {
+        let indentInsertedNewLineCharactor = "\n" + String(repeating: " ", count: "\(source.0): ".count)
+        let indentInserted = source.1.replacingOccurrences(of: "\n", with: indentInsertedNewLineCharactor)
+        return "\(source.0): \(indentInserted)"
```

Comment:
Can you use `String#indentTail` instead to more simply?

e.g.

```swift
source.1.indentTail(size: "\(source.0): ".count) // I'm not confirm to this :)
```

Reply:
Using `indentTail()` passed tests! thx suggestion! 8fe61d51d35a459069dfdcf80431ad7a0e331978